OPHD/Things/Structures/Residence.h

```diff
@@ -29,17 +25,8 @@ class Residence : public Structure
 
 	int capacity() const { return mCapacity; }
 
-	void drawInspectorView(NAS2D::Renderer& renderer, const NAS2D::Rectangle<float>& windowRect) override
+	void drawInspectorView([[maybe_unused]] NAS2D::Renderer& renderer, const NAS2D::Rectangle<float>& windowRect) override
```

Comment:
The `[[maybe_unused]]` here makes me a bit apprehensive. It's theoretically possible there could be two distinct rendering surfaces (this game just isn't setup to function that way). I feel like the `Renderer` should be passed through to the text rendering functions. Either that, or we omit it here to ensure consistency.

I made a suggestion to Leeor a little while back that we start passing `Renderer` parameters around explicitly in a number of places, such as to control draw functions (and that `update` and `draw` be separate methods). My preference is to use explicit parameters over `Utility`/global variables.

Reply:
I'd like to just pull renderer out of the drawInspectorView for now. I was thinking this as the right course after pushing the PR.

Generally agree best practice is to keep Update and Draw logic separate. Certainly could be uses for multiple renderers and I have typically passed them as suggested.